samples/wifi/thread_coex/README.rst

```diff
@@ -0,0 +1,397 @@
+| coexistence disabled   |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Thread,      | 8.8                | 63                 |
+| coexistence enabled    |                    |                    |
++------------------------+--------------------+--------------------+
+
+
+Wi-Fi (802.11n mode) in 5 GHz
+-----------------------------
+
+Wi-Fi in client role (channel 48), Thread in client role (channel 22):
+
++------------------------+--------------------+--------------------+
+| Test case              | Wi-Fi UDP Tx       | Thread             |
+|                        | throughput in Mbps | throughput in kbps |
++========================+====================+====================+
+| Wi-Fi only             | 9.6                | N.A                |
+|                        |                    |                    |
++------------------------+--------------------+--------------------+
+| Thread only            | N.A                | 63                 |
+|                        |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Thread,      | 9.2                | 63                 |
+| coexistence disabled   |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Thread,      | 9.1                | 63                 |
+| coexistence enabled    |                    |                    |
++------------------------+--------------------+--------------------+
+
+Wi-Fi in client role (channel 48), Thread in server role (channel 22):
+
++------------------------+--------------------+--------------------+
+| Test case              | Wi-Fi UDP Tx       | Thread             |
+|                        | throughput in Mbps | throughput in kbps |
++========================+====================+====================+
+| Wi-Fi only             | 9.6                | N.A                |
+|                        |                    |                    |
++------------------------+--------------------+--------------------+
+| Thread only            | N.A                | 63                 |
+|                        |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Thread,      | 8.6                | 63                 |
+| coexistence disabled   |                    |                    |
++------------------------+--------------------+--------------------+
+| Wi-Fi and Thread,      | 8.5                | 63                 |
+| coexistence enabled    |                    |                    |
++------------------------+--------------------+--------------------+
+
+As is evident from the results of the sample execution, coexistence harmonizes air-time between Wi-Fi and Thread rather than resulting in a higher combined throughput.
```

Comment:
```suggestion
As is evident from the results of the sample execution, coexistence harmonizes air-time between Wi-Fi and Thread rather than resulting in a higher combined throughput.
```
```suggestion
The results show that coexistence harmonizes airtime between Wi-Fi and Thread rather than resulting in a higher combined throughput.
```